examples/example_full_kinodynamic_walking/main_rotate.cpp

```diff
@@ -178,13 +178,13 @@ int main(int argc, char** argv) {
   auto centroid = Point3(0, 0, 0);
   for (auto&& link : links)
     centroid +=
-        (link_map[link]->wTcom() * Pose3(Rot3(), c0.point)).translation();
+        (link_map[link]->bTcom() * Pose3(Rot3(), c0.point)).translation();
```

Comment:
Not strictly the goal of this PR, but might as well change all of these to `bTcom().transform_from(c0.point())` , which can be done using the overloaded `*` operator, hence:
``` 
centroid += link_map[link]->bTcom() * c0.point();
```

Reply:
done